Formula/gitoxide.rb

```diff
@@ -0,0 +1,28 @@
+class Gitoxide < Formula
+  desc "Idiomatic, lean, fast & safe pure Rust implementation of Git"
+  homepage "https://github.com/Byron/gitoxide"
+  url "https://github.com/Byron/gitoxide/archive/refs/tags/v0.25.0.tar.gz"
+  sha256 "098bb18e1cae42ab7597b6b442538d3f51b57935a848ea121e20e2921d6a4693"
+  license "Apache-2.0"
+
+  # Cmake is here as only rust install wasn't enough
```

Comment:
Why wasn't it enough?

Reply:
Pushed a change that I _think_ will allow us to get rid of CMake and use system Zlib instead.